pkg/controller/command/sdscomm/models.go

```diff
@@ -8,18 +8,21 @@ package sdscomm
 
 import "encoding/json"
 
-type DIDDocData struct {
-	Name     string          `json:"name,omitempty"`
-	SignType string          `json:"signType,omitempty"`
-	DID      json.RawMessage `json:"did,omitempty"`
+type SaveDIDDocToSDSRequest struct {
+	Name          string          `json:"name,omitempty"`
+	SignType      string          `json:"signType,omitempty"`
+	DID           json.RawMessage `json:"did,omitempty"`
+	AgentUsername string          `json:"agentUsername,omitempty"`
```

Comment:
may be a followup to have better name, 'userid, username, user etc', term 'agent' is misleading. Same applies to other requests in this file.

Reply:
Updated to "userID".